src/cookbooks/clients_last_seen_bits.md

```diff
@@ -57,7 +57,7 @@ ORDER BY
   submission_date
 ```
 
-Under the hood, `days_since_seen` is calculated using the `bits28_days_since_seen`
+Under the hood, `days_since_seen` is calculated using the `bits28.days_since_seen`
```

Comment:
I think we should fully qualify the names everywhere for consistency. Some of the SQL examples here still won't work because the lack the `mozfun.` prefix.

Reply:
I take it back. I think all the actual examples here are now valid 👍

Reply:
Yeah, I wasn't sure about putting the `mozfun` prefix on instances that just were describing the functions (not examples). I ended up keeping the existing structure, which didn't have the dataset names